module/main.go

```diff
@@ -0,0 +1,39 @@
+// Package main provides an ORB_SLAM3 implementation of a SLAM module
+package main
+
+import (
+	"context"
+
+	"github.com/edaniels/golog"
+	"go.viam.com/rdk/module"
+	"go.viam.com/rdk/services/slam"
+	"go.viam.com/utils"
+
+	viamorbslam3 "github.com/viamrobotics/viam-orb-slam3"
+)
+
+func main() {
+	utils.ContextualMain(mainWithArgs, golog.NewLogger("orbslam3Module"))
+}
+
+func mainWithArgs(ctx context.Context, args []string, logger golog.Logger) error {
+	// Instantiate the module
+	orbModule, err := module.NewModuleFromArgs(ctx, logger)
+	if err != nil {
+		return err
```

Comment:
Under what conditions will this error?

Reply:
@nicksanford we may not know; I believe this is following the pattern suggested for all modular resources: https://docs.google.com/presentation/d/15lLqMSZMW0DguVcAs0mhH0mDBOwQysXRmTJO-x9j964/edit#slide=id.g1baaa848a7e_0_102

Reply:
I confirmed that if an error is returned from `mainWithArgs` that the module process exists with status code 1 & that the termination is logged in RDK:
```
2023-04-05T11:23:25.486-0400    ERROR   robot_server.modmanager.process.orb_slam3_module_/usr/local/bin/orb-slam3-module        pexec/managed_process.go:274    error waiting for process during manage       {"error": "exit status 1"}

```

Reply:
Currently, as written this never errors.